source/settings/command-line-options.txt

```diff
@@ -96,6 +103,10 @@ value in the :guilabel:`Settings` panel.
      - Enable :ref:`automatic updates <enable-automatic-updates>`. Use 
        ``--no-autoUpdates`` to disable automatic updates. 
 
+   * - .. option:: --browserCommandForOIDCAuth
+     - Specify a shell command to start the browser for OIDC authentication 
+       when connecting to the server or logging into your Atlas Cloud account.
```

Comment:
[suggestion] can we make this active voice? (I'm not sure if this sentence is supposed to mean the user is connecting to the server or if MongoDB is so my suggestion might not make sense in this context)

```suggestion
       when you connect to the server or log into your Atlas Cloud account.
```

Reply:
updated.